File: app/src/main/java/com/samkeet/takeup/fragments/LoginFragment.java
package com.samkeet.takeup.fragments;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.samkeet.takeup.R;
import com.samkeet.takeup.activities.LoginActivity;
import com.samkeet.takeup.activities.SignupActivity;

/**
 * Created by Hi Abhishek on 5/9/2017.
 */

public class LoginFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_login, container, false);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        Button login = (Button) view.findViewById(R.id.login_button);
        Button signup = (Button) view.findViewById(R.id.signup_button);

        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getContext(), LoginActivity.class);
                startActivity(intent);
            }
        });

        signup.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getContext(), SignupActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/samkeet/takeup/fragments/LoginFragment.kt
package com.samkeet.takeup.fragments

import android.content.Intent
import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.samkeet.takeup.R
import com.samkeet.takeup.activities.LoginActivity
import com.samkeet.takeup.activities.SignupActivity
import kotlinx.android.synthetic.main.fragment_login.*

/**
 * Created by knightshade on 10/19/17.
 */
class LoginFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.fragment_login, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {

        btn_login.setOnClickListener {
            startActivity(Intent(context, LoginActivity::class.java))
        }

        btn_signup.setOnClickListener {
            startActivity(Intent(context, SignupActivity::class.java))
        }
    }
}